src/renderer/features/contacts/CreateContact/model/contact-form.ts

```diff
@@ -0,0 +1,97 @@
+import { attach, createApi, createStore, sample } from 'effector';
+import { createForm } from 'effector-forms';
+import { not, or } from 'patronum';
+
+import { Contact, contactModel } from '@renderer/entities/contact';
+import { toAccountId, validateAddress } from '@renderer/shared/lib/utils';
+import { validateFullUserName } from '@renderer/shared/api/matrix';
+
+export type FormApi = {
+  onSubmit: () => void;
+};
+const $formApi = createStore<FormApi | null>(null);
```

Comment:
Pass callbacks to model, so it can execute them after form submit
Similar [approach](https://withease.pages.dev/magazine/handle_events_in_ui_frameworks.html) for api calls (provided by effector team)